plugin/wakatime.vim

```diff
@@ -90,13 +95,16 @@ let s:VERSION = '9.0.1'
         " Detect os and architecture
         if s:IsWindows()
             let s:osname = "windows"
-            if has("win64")
+            if has("win64") || isdirectory(expand('$WINDIR') . '\SysWOW64')
```

Comment:
Let's be extra careful and do:

`if has("win64") || exists('$WINDIR') && isdirectory(expand('$WINDIR') . '\SysWOW64')`

Reply:
Ok. I don't know if there is any other better way to detect 64-bit.